mmv1/templates/terraform/examples/network_security_client_tls_policy_basic.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_network_security_client_tls_policy" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
The resource name should be google_network_services_security_client_tls_policy as it is a product in the service networkservices.

Reply:
The product is in network security

Reply:
Oh. Yes, you are right. Sorry about that.